src/classes/Settings.js

```diff
@@ -109,6 +109,7 @@ class Settings {
       is_government: configContent.is_government,
       facebook_pixel: configContent["facebook-pixel"],
       google_analytics: configContent.google_analytics,
```

Comment:
guessing this is the old GA tag that we will deprecate after a while?

Reply:
Yup, we'll be deprecating this after July when google removes support for the old google analytics (UA)

Reply:
got it